do not remove the OK button

There's no hsep. The ugly hack removed the necessary OK button.

Index: system-config-keyboard-1.3.1/src/keyboard_gui.py
===================================================================
--- system-config-keyboard-1.3.1.orig/src/keyboard_gui.py
+++ system-config-keyboard-1.3.1/src/keyboard_gui.py
@@ -254,10 +254,6 @@ class moduleClass(Module):
         okButton = self.mainWindow.add_button('gtk-ok', 0)
         okButton.connect("clicked", self._okClicked)
 
-        # Remove the hsep from the dialog.  It's ugly.
-        hsep = self.mainWindow.get_children()[0].get_children()[0]
-        self.mainWindow.get_children()[0].remove(hsep)
-
         self.mainWindow.vbox.pack_start(self.vbox)
         self.mainWindow.show_all()
 

